kombu/transport/pubsub.py

```diff
@@ -212,10 +221,22 @@ def _msg_get(self, subscription_path):
             for msg in resp.received_messages:
                 if self.temp_cache[subscription_path].full():
                     break
+                data = loads(msg.message.data)
+                # Check messages type, ignore if unexpexted
+                if not isinstance(data, dict):
+                    logger.warning("".join([
+                        "Pulled message of unexpected type: ",
+                        str(type(data)), " data: ", data]))
+                    # Ack it
+                    self.subscriber.\
```

Comment:
was this the approach discussed? to ack unexpected message?

Reply:
@jeechu what do you plan to do with the message instead ?

Reply:
@mpsmks @avinashdunzo any suggestions?

Reply:
for go services, we have not acked any random message, lets do that here too
lets add a log and not ack it

Reply:
changes done